scripts/DGSPlanner/DGSPlannerGUI.py

```diff
@@ -10,19 +10,12 @@
 from . import ClassicUBInputWidget
 from . import MatrixUBInputWidget
 from . import DimensionSelectorWidget
-from PyQt4 import QtCore, QtGui
+from qtpy import QtCore, QtGui,QtWidgets
 import sys
 import mantid
-import mantidqtpython as mqt
 from .ValidateOL import ValidateOL
-from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
-try:
-    from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT as NavigationToolbar
-except ImportError:
-    try:
-        from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg as NavigationToolbar
-    except ImportError:
-        raise ImportError("Cannot import navigation toolbar")
+import matplotlib
+from MPLwidgets import *
```

Comment:
Maybe just import what you need.

```suggestion
from MPLwidgets import FigureCanvas, NavigationToolbar2QT
```